### DISCUSSION TITLE
Hi Mew Tracker I need help understanding your formula

### DISCUSSION THREAD

**Discussion by nkcr888** [kwsch/PKHeX#3703] [Q&A]
Hi Mew. Just as said before there isnt a guide out there for changing the daily spawns in DPP or BDSP for PKHex so I was wondering how I could use the formula you place to put in the right values in PKHex. If you could help me with this I'd be very grateful. I'm not very code or formula savvy.

**Comment by Lusamine** [comment]
I moved this to discussions. This isn't an issue with PKHeX, but a question about usage.

**Reply by nkcr888** [reply]
Thanks Anubis.

**Comment by kwsch** [comment]
https://projectpokemon.org/home/forums/topic/60922-pokemon-dp-great-marsh-swarm-trophy-garden-finds/

**Reply by nkcr888** [reply]
Appreciate this resource but I cant understand how to parce it as I dont know to to manipulate block data. But having the swarm values are a great start into understanding this and using it on the current build of PKHex

**Reply by kwsch** [reply]
> Appreciate this resource but I cant understand how to parce it as I dont know to to manipulate block data. But having the swarm values are a great start into understanding this and using it on the current build of PKHex

Change the {seed} to whatever value calculates the species you want, per the details listed in that thread.

**Reply by nkcr888** [reply]
How do you change the seed in the current version of PKHex?

**Reply by kwsch** [reply]
Block Editor button will bring up the full PropertyGrid exposing all savefile objects and documented properties. Find the safari seed and change it to whatever you want.
